server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -762,6 +771,21 @@ public String serviceName(ServiceRequestContext ctx) {
         }
     }
 
+    @Nullable
+    private static Function<? super HttpService, CorsService> configCorsDecorator(CorsConfig corsConfig) {
+        if (corsConfig == null || corsConfig.allowedOrigins() == null ||
+            corsConfig.allowedOrigins().isEmpty()) {
```

Comment:
I thought `corsConfig.allowedOrigins` couldn't be `null` nor `empty` due to validations in the constructor of `corsConfig`

```suggestion
        if (corsConfig == null) {
```